IPython/core/magics/history.py

```diff
@@ -289,8 +289,10 @@ def rerun(self, parameter_s=''):
           current command.
 
           -g foo : Repeat the most recent line which contains foo
+
+          -q     : Quiet mode: do not echo the cell contents back
```

Comment:
I like this change, but could you make it clearer in the help docs that `quiet` means to suppress the history lines specifically? A user will still see any output from the command, right? "Quiet" usually implies that all regular output will be suppressed (i.e. non-errors) or maybe just lower verbosity.

Reply:
Documentation tone can often be the hardest part of a contribution. I've reworded this, but let me know if it's still not quite right, or feel free to specify wording yourself if you'd like.